src/doc/contrib/src/team.md

```diff
@@ -0,0 +1,157 @@
+  - Development of a previously approved unstable feature that matches the expected development of that feature.
+
+- Small features or changes, large refactorings, or major changes to Cargo's codebase or process require an approval by the team via consensus.
+  These decisions can be done via the FCP process of [rfcbot], or in an ad-hoc manner such as during a team meeting.
+  rfcbot FCP requests do not require waiting for the 10-day feedback window if there is a complete team consensus, as this process is mostly aimed at polling the team rather than publicly soliciting feedback.
+  Though, public feedback is welcome at any time.
+
+  Some examples of what this might cover are:
+  - Addition of a new, minor command-line argument, or an addition of an option to an existing one.
+  - Addition of new fields and values to JSON outputs.
+  - A bug fix or change that may technically involve a backwards-incompatible change.
+    See the [Backwards compatibility] section for some examples.
+  - Documentation changes that may substantially change the expected usage of Rust and Cargo.
+    For example, the [SemVer chapter] contains subjective prescriptions for how users should develop their code.
+  - A significant change in Cargo's console output.
+  - A significant change to Cargo's code structure, or how maintenance or usage of the Cargo codebase is handled.
+  - Beta backports that are risky or have any uncertainty about their necessity.
+  - [Stable backports].
+    These usually also require involvement with the Release team.
+  - A significant change to the management of the Cargo team itself or the processes it uses, such as significant updates to this document.
+  - Addition of new members to the Cargo team, or other actions involving the team membership.
+    These decisions are usually processed via private channels by the entirety of the team.
+  - A change that is a "one-way door".
+    That is, something that is difficult to reverse without breaking backwards compatibility.
+
+- Larger features should usually go through the [RFC process].
+  This usually involves first soliciting feedback from the Cargo team and the rest of the community, often via the [Rust Internals] discussion board, [Cargo's issue tracker], and the [Zulip] channel.
+  If there is positive feedback to the idea, the next step is to formally post an RFC on the RFC repo.
+  The community and the Cargo team will have an opportunity to provide feedback on the proposal.
+  After some period of time, the Cargo team may decide to either accept, postpone, or close a proposal based on the interest in the proposal and the team's availability.
+
+  Some examples of what this might cover are:
+  - Major changes or new features or options in `Cargo.toml` or the config files.
+  - Changes to the registry index or API.
+  - New or changed CLI options that are expected to have a significant impact on how Cargo is used.
+  - New `cargo` commands.
```

Comment:
We have added some commands with a change proposal, rather than an FCP.  Should we make a distinction on this?

I'm assuming RFC is more needed when the command has non-trivial workflows and hasn't been broadly adopted by the community to vet those workflows. 
- `cargo rm` is more trivial
- `cargo add` is a little less trivial but had broad use for a long time
- `cargo-script` has non-trivial workflows and isn't pervasive.  Even if it was, the workflows are non-trivial enough to warrant an RFC to re-evaluate those design decisions
  - I think `cargo-nextest` is a good example of this also.  It has done a lot to improve workflows but in a way that works around limitations in the ecosystem.  Instead of replacing `cargo test` with it, we'd need to decide what non-workaround solutions to the problems look like

Reply:
Yea, this is a bit of a gray area.  I added some extra text here, though I'm not sure if it is really complete. Let me know if you have any suggestions to the wording.